src/app/community/community.component.ts

```diff
@@ -51,7 +55,10 @@ export class CommunityComponent implements OnInit {
   }
 
   ngOnInit() {
+    var urlFragment = this.router.url.split('/')[2];
```

Comment:
I think it's better to use `let` instead of `var`

Reply:
There is also a lint rule for this (see other comment)